H/HelFEM/build_tarballs.jl

```diff
@@ -0,0 +1,83 @@
+using BinaryBuilder, Pkg
+
+name = "HelFEM"
+version = v"0.0.1"
+sources = [
+    GitSource("https://github.com/mortenpi/HelFEM.git", "97467946bb44c970c94436ed896840227e4b599f")
```

Comment:
I had an overzealous `.gitignore` rule, so one of the necessary files didn't get committed.

```suggestion
    GitSource("https://github.com/mortenpi/HelFEM.git", "aeb49aad6319094f3129c3d81e4e9c7431d95d76")
```